easy/Challange6.js

```diff
@@ -0,0 +1,24 @@
+//  Create a function that takes an array of numbers and return both the minimum and maximum numbers, in that order.
+
+let a=[18,2,45,4,15,6,77,8];
```

Comment:
It should be `const`

Reply:
updated 
Commit Id 666b31ef75580e7dc0114c4e0ed0bdf0fbf45963